apps/frontend/src/pages/CreateCharity.tsx

```diff
@@ -0,0 +1,671 @@
+        index: index,
+        image: event.target.files?.[0],
+        imageString: imageString,
+      };
+      setPartnerImages((prevState) => [...prevState, newImage]);
+    }
+  };
+
+  // Get the org image from the given index
+  const getImageFromOrgIndex = (index: number) => {
+    for (let i = 0; i < partnerImages.length; i++) {
+      if (partnerImages[i].index == index) {
+        return partnerImages[i].imageString;
+      }
+    }
+    return undefined;
+  };
+
+  // Converts the image file blob to base64 for preview
+  const imageBlobToBase64 = (
+    image: File | null,
+  ): Promise<string | undefined> => {
+    return new Promise((resolve, reject) => {
+      if (image) {
+        const reader = new FileReader();
+        reader.onload = () => {
+          resolve(reader.result as string);
+        };
+        reader.onerror = (error) => {
+          reject(error);
+        };
+        reader.readAsDataURL(image);
+      } else {
+        resolve(undefined);
+      }
+    });
+  };
+
+  return (
+    <>
+      <SearchBar />
+      <Container>
+        <Card sx={{ marginTop: "32px", padding: "10px" }}>
+          <CardContent>
+            <Typography variant={"h5"}>Create Charity</Typography>
+            <Box>
+              <form noValidate autoComplete="off" onSubmit={handleSubmit}>
+                <FormControl
+                  sx={{ width: "100%", padding: "10px", gap: "10px" }}
+                >
+                  <TextField
+                    type="text"
+                    label="Charity Title"
+                    id="charity-title"
+                    onChange={updateCharityName}
+                    value={newCharityObject.name}
+                  />
+                  <TextField
+                    type="text"
+                    multiline
+                    label="Charity Description"
+                    id="charity-description"
+                    onChange={updateDescription}
+                    value={newCharityObject.description}
+                  />
+                  <img
+                    src={logoImageString}
+                    style={{
+                      display: logoImageString ? "block" : "none",
+                      maxHeight: "300px",
+                      width: "auto",
+                      aspectRatio: "auto",
+                      objectFit: "contain",
+                    }}
+                    alt={"Charity Event Logo"}
+                  />
+                  <Button
+                    variant="contained"
+                    component="label"
+                    id="upload-logo-button"
+                  >
+                    Upload Picture
+                    <input
+                      id="upload-logo-input"
+                      type="file"
+                      hidden
+                      accept="image/*"
+                      onChange={updateCharityImage}
+                    />
+                  </Button>
+                  <TextField
```

Comment:
How we're not assigning an end date?

Reply:
We definitely could, but aren't they supposed to be only a month long? I had it in here originally, then thought that maybe I don't need to have it since I can just add 31 days to the start date, would love to hear your thoughts though!

Reply:
Seemed Scott wanted it so I just added that field back